backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/HybridBackupCommand.java

```diff
@@ -169,6 +169,8 @@ public boolean performNextOperation(int completedChildCount) {
     @Override
     protected void endSuccessfully() {
         updateVmBackupPhase(VmBackupPhase.SUCCEEDED);
+        getParameters().getVmBackup().setSnapshotId(null);
+        vmBackupDao.update(getParameters().getVmBackup());
```

Comment:
wouldn't it better to set the snapshot id as a foreign key with on delete cascade instead?

Reply:
the problem is that backups remain in the DB after completion (for 15 minutes by default), so cascade can cause problems, but I now learned about `on delete set null` and I'll give it a try :)

Reply:
> the problem is that backups remain in the DB after completion (for 15 minutes by default), so cascade can cause problems, but I now learned about `on delete set null` and I'll give it a try :)

yeah, I wrote "on delete cascade" but actually meant that :)